src/evm-config.js

```diff
@@ -39,6 +39,16 @@ function filenameToConfigName(filename) {
   return match ? match[1] : null;
 }
 
+function ensureConfigExists(name) {
```

Comment:
_(nit)_ Could this be renamed to something else, e.g. `testConfigExists()` I usually see 'ensureThing()' used  as "if thing is false, then make thing true" -- so to me this function name reads "if name doesn't exist, then create it"

Reply:
Yep that makes sense.